### PR TITLE
[DDW-966] Improve RTS splash message

### PR DESCRIPTION
**Pull request by przemyslaw-wlodek**
This PR adds better description to the RTS splash screen message. It includes instruction how to enable/disable RTS flags.

## Screenshots

image
image

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1646061063178959)
- [ ] Test

---

### Basics

- [x] PR assigned to the PR author(s)
- [x] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [x] If there are UI changes, Alexander Rukin assigned as an additional reviewer
- [ ] All visual regression testing has been reviewed (assign `run Chromatic` label to PR to trigger the run)
- [x] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [x] PR link is added to a Jira ticket, ticket moved to In Review
- [x] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [x] PR has a good description that summarizes all changes
- [x] PR contains screenshots (in case of UI changes)
- [x] CHANGELOG entry has been added to the top of the appropriate section (_Features_, _Fixes_, _Chores_) and is linked to the correct PR on GitHub
- [x] There are no missing translations (running `yarn manage:translations` produces no changes)
- [x] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [x] Japanese text changes are proofread and approved (Junko Oda)
- [x] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality

- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with typescript types
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing

- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review

- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/renderer/app/components/knownIssues/RTSFlagsRecommendationOverlay/RTSFlagsRecommendationOverlay.scss [hunk 1]
```diff
@@ -96,6 +96,14 @@
         background-color: rgba(45, 45, 45, 0.5);
       }
     }
+
+    & p {
+      margin-bottom: 11px;
+
+      &:last-child {
+        margin-bottom: 0;
```
Comment (alexander-rukin) [inline source/renderer/app/components/knownIssues/RTSFlagsRecommendationOverlay/RTSFlagsRecommendationOverlay.scss:104]: isn't "p + p" work anymore?

**Review by alexander-rukin — APPROVED** [review]
looks ok

**Review by marcin-mazurek — APPROVED** [review]
image

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 ### Features
 
 - Implemented error message when trying to leave wallet without enough ada to support tokens ([PR 2783](https://github.com/input-output-hk/daedalus/pull/2783))
+- Improved RTS flags splash screen message ([PR 2901](https://github.com/input-output-hk/daedalus/pull/2901))
```
Comment (szymonmaslowski) [inline CHANGELOG.md:8]: Please move this line to the top of the section

File: storybook/stories/news/AlertsOverlay.stories.tsx [hunk 3]
```diff
@@ -108,4 +110,7 @@ storiesOf('News|Overlays', module)
         dateOptionsIntl[props.locale][0].value
       )}
     />
+  ))
+  .add('RTS Recommendation', () => (
+    <RTSFlagsRecommendationOverlay onConfirm={noop} onClose={noop} />
```
Comment (szymonmaslowski) [inline storybook/stories/news/AlertsOverlay.stories.tsx:115]: Please use the `action` from storybook actions plugin ;)

**Review by szymonmaslowski — CHANGES_REQUESTED** [review]
Just have two small recommendations

**Review by ManusMcCole — APPROVED** [review]
Looks good to me 👍

**Pull request merged**